Controllers/UserController.cs

```diff
@@ -111,4 +207,4 @@ public IActionResult RoleTest1()
             return Ok(HttpContext.User.Identity.Name);
         }
     }
-}
+}
```

Comment:
Why is newline removed?

Reply:
This line wasn't need (empty unusuable space).

Reply:
It is highly recommended to put newline at the end of each file for readability. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline